ci-operator/jobs/openshift/library/openshift-library-master-periodics.yaml

```diff
@@ -46,7 +46,7 @@ periodics:
           secretKeyRef:
             key: oauth
             name: github-credentials-openshift-merge-robot
-      image: quay.io/bparees/importer:latest
+      image: registry.svc.ci.openshift.org/openshift/release:golang-1.13
```

Comment:
Referencing `registry.svc.ci.openshift.org` images directly is a no-no.

@stevekuznetsov What should we reference here?

Reply:
perhaps `openshift/release:golang-1.13`? Will the imagestream resolve correctly?

Reply:
This should just always use the local clusters registry, is there same vanity url for that (that works in both 3.11 and 4X)?